crates/cast/bin/args.rs

```diff
@@ -539,6 +539,16 @@ pub enum CastSubcommand {
         data: String,
     },
 
+    /// Decode custom error data.
+    #[command(visible_aliases = &["error-decode", "--error-decode", "erd"])]
+    DecodeError {
+        /// The error signature. If none provided then tries to decode from local cache.
+        #[arg(long = "sig", visible_alias = "error-sig")]
```

Comment:
`long = "sig"` should not be necessary here I think

Reply:
yeah, I want to have sig non required but able to be specified before data, if not adding long it fails with `Found non-required positional argument with a lower index than a required positional argument: "sig" index Some(1)` is there a different way than workaround it by `long = "sig"`?

Reply:
ah, nm, I will only add `long`

Reply:
added in https://github.com/foundry-rs/foundry/pull/9428/commits/d349c9a3cbf22e65db5670ed3d047712926642ea